src/components/PostList.jsx

```diff
@@ -7,13 +7,26 @@ const PostList = ({
   handleLikeCount,
   openCommentBox,
   postList,
+  onSubmitComment,
+  onDeleteComment,
 }) => {
   const { name } = useContext(UserContext);
+  const [writeComment, setWriteComment] = useState("");
+
+  const submitComment = (id) => {
+    writeComment && onSubmitComment(writeComment, id);
+    setWriteComment("");
+  };
```

Comment:
Let's avoid short circuiting while we can. Writing maintainable code is hard. So we should always target simplicity.
```suggestion
  const submitComment = (id) => {
    if(comment.trim() === ""){
       return
    }
    
    onSubmitComment(writeComment, id);
    setComment("");
  };
```